doc/create_ramp_event.rst

```diff
@@ -56,25 +56,33 @@ This config file should look like::
 
 Before you continue make sure that:
 
-    1.  you have the private and the public data in the following directory
-        structure::
+    1.  your private and public data are available. For example, you might have
```

Comment:
maybe:
'Your private and public data are in `ramp-kits/<my_challenge/data` and `ramp-data/<my_challenge/data` respectively.